resources/assets/js/helpers/router.tsx

```diff
@@ -70,3 +70,20 @@ export const navigate = (url: string): void => {
 export const redirect = (url: string): void => {
   HISTORY.replace(url);
 };
+
+export const Link: React.FC<{ href: string; title: string }> = ({
+  href,
+  title,
+  children,
+}): React.ReactElement => (
```

Comment:
@tristan-orourke couple changes here as well. Our linter was complaining about the `url` prop, saying the anchor did not have a valid `href` so I just used the same variable for both. I also condensed the return, if you're just returning a fragment/tag, you can forego the `return` keyword and just return the tag directly.